pkg/scheduler/metrics/metrics.go

```diff
@@ -52,6 +60,18 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	PreemptionVictims = prometheus.NewGauge(
```

Comment:
I am not sure if this is useful when we have `pod_preempted`. These two should be almost always the same value. It would be good if we have the number of times that preemption is attempted. In other words, the number of times that we call `preempt`. So, I'd replace this with the number of preemption attempts.

Reply:
> I am not sure if this is useful when we have pod_preempted. These two should be almost always the same value.

You are right but I wasn't quite sure about terminology so added, I will add a metric for no of preemptions attempted now.